scripts/wattlab_parity_oracle_dump.py

```
@@ -53,6 +85,103 @@ def _maybe_reexec_vibe19_venv(vibe19_root: Path) -> None:
+    c_pkg = "/data/raw_BUILDING_100_openfdd.zip"
+    c_sched = "/data/parity/fixtures/schedule_b100_7to5.json"
+    c_out = "/data/parity/artifacts/vibe19_oracle"
+    if package.resolve() != DEFAULT_PKG.resolve():
+        print(
+            f"warning: docker dump uses bind {c_pkg}; --package {package} ignored",
+            file=sys.stderr,
+        )
+    argv = _afdd_argv(c_pkg, c_out, c_sched, profile, skip_rules)
+    cmd = ["docker", "exec", container, "python", "scripts/agent_afdd.py", *argv]
+    print("docker:", " ".join(cmd), flush=True)
+    proc = subprocess.run(cmd)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)